packages/base/src/mainview/mainview.tsx

```diff
@@ -386,7 +408,21 @@ export class MainView extends React.Component<IProps, IStates> {
     sender: IJupyterGISDoc,
     change: MapChange
   ): void {
-    // TODO SOMETHING
+    if (!this._initializedPosition) {
+      const options = this._model.getOptions();
+
+      // It is important to call setZoom first, otherwise maplibre does set the center properly
+      this._Map.setZoom(options.zoom || 0);
+      this._Map.setCenter(
+        (options.longitude &&
+          options.latitude && {
+            lng: options.longitude,
+            lat: options.latitude
+          }) || [0, 0]
+      );
+
+      this._initializedPosition = true;
```

Comment:
If the options are empty in the doc, I think this method is never called and the the `_initilaizedPosition` remains false.

Reply:
Right thanks! 

I fixed the untitled file creation to fix missing `options` entry. I guess we should consider the file malformed if the options are not there, I changed the schema accordingly.